distribution/docs/src/main/resources/content/_configuring/exporting-existing-configurations-contents.adoc

```diff
@@ -32,7 +32,8 @@ image::exporting_configuration_step3.png[Exporting Step 3]
 
 To export the current ${branding} configuration from the ${command-console}:
 
-. Enter `migration:export <directory>`. This command creates the exported configuration file `exported-${project.version}.zip` which is saved to the specified directory. If no directory is specified it will default to `exported` directory underneath `<${branding}_HOME>`.
+. Run the `migration:export` command from the ${command-console}.
```

Comment:
✏️ minor nitpick, but would it sound better to say "Run the command `migration:export`... ? It sounds more like "here is the exact thing to type."